docs/operatorTestPlaybooksCI.md

```diff
@@ -0,0 +1,52 @@
+
+# Usage
+* The [test-operator-test-playbooks](https://jenkins-cvp-ci.cloud.paas.psi.redhat.com/job/test-operator-test-playbooks/) job runs on the CVP CI Jenkins server.
+
+* The job is triggered by the Fedora Messaging messages which:
+
+  * Indicate that a pull request was opened, re-opened, or synchronized (“synchronized” meaning that code changes were pushed to an already-opened pull request)
+
+    Ex: [PR-opened](https://apps.fedoraproject.org/datagrepper/raw?topic=org.fedoraproject.prod.github.pull_request.opened)
+
+  * Indicate a comment was added that contains the text `/retest` or `/test CVP`
+
+    Ex: [PR-comment-added](https://apps.fedoraproject.org/datagrepper/raw?topic=org.fedoraproject.prod.github.issue.comment&delta=3600)
+
+* The openshift project for the `test-operator-test-playbooks` is `cvp-ci-operatortestplaybooks`
+
+* The test-operator-test-playbooks job parses the fedmsg to determine what files have changed and what repository and branch need to be tested.
+
+* If the data can be successfully parsed, test-operator-test-playbooks uses `CVP bot account` on GitHub to create a check status, `CVP/pr-sanity-test`, for the pull request.
+
+* Note: If the only file that has changed is README.md, test-operator-test-playbooks will then exit successfully. No further tests are required since no ansible playbooks have changed.
+
+* The test-operator-test-playbooks job next configures it’s own [Team Jenkins server](https://jenkins-cvp-ci-operatortestplaybooks.cloud.paas.psi.redhat.com/) to run the testing pipelines.
+
+* When this is complete the test-operator-test-playbooks currently triggers three test pipelines on it’s team Jenkins server:
+
+  * cvp-redhat-operator-bundle-image-validation-test
+  * cvp-isv-operator-metadata-validation-test
+  * cvp-redhat-operator-metadata-validation-test
+
+* These are the same pipelines that run in production. The operators used for testing in these pipelines are the same used in [CVP-CI setup tests](https://gitlab.sat.engineering.redhat.com/cvp/pipeline/-/tree/master/ci/tests/setupTests/sample-ci-messages).
+
+* The pipelines are running in parallel, although they are started one minute apart from each other. 
+
+* When a testing pipeline is started, the test-operator-test-playbooks job adds a new check status to the pull request on GitHub indicating the name of the test pipeline and it sets the status to “pending”, so that end-users are informed that a test is in progress.
+
+* When a testing pipeline completes, the check status is updated with either `success` or `failure`.
+
+* Each pipeline’s artifacts are uploaded to Amazon S3, the same artifacts as are uploaded in production, and can be viewed externally to determine which stages in the testing pipeline failed. 
+
+  * [Example coldstorage URL](http://external-ci-coldstorage.datahub.redhat.com/cvp/cvp-redhat-operator-bundle-image-validation-test/e2e-test-operator-container-8.0-367/2b303e1a-a4a9-4815-b604-44d67c81dc64/) 
```

Comment:
There should not be any internal links in upstream documentation.
It is unfair for the community.